lang/ilias_en.lang

```diff
@@ -1275,6 +1275,7 @@ assessment#:#tst_add_quest_cont_edit_mode_RTE_info#:#Allows text formatting of q
 assessment#:#tst_addit_passes_blocked_after_passed_msg#:#You have passed the test. The test cannot be started again.
 assessment#:#tst_all_test_competences#:#All Test Competence
 assessment#:#tst_all_user_data_deleted#:#All user data for this test has been removed!
+assessment#:#tst_already_passed_cannot_retake#:#Test already passed. You cannot retake the test.
```

Comment:
Hier wäre auch konsistenter "You cannot start the again."

Reply:
Ist entsprechend angepasst.